src/pages/Router/Router.tsx

```diff
@@ -116,6 +126,9 @@ const ZdruzenieRouter: React.FC = () => {
       <Route exact path={path}>
         <PagePlaceholder title="Združenie home" />
       </Route>
+      <Route exact path="/register/">
```

Comment:
maju tu byt dve lomky? resp., asi to ma byt rovnako ako ostatne, ze `path + 'register/'`?